packages/react-dom/src/__tests__/ReactFunctionComponent-test.js

```diff
@@ -359,7 +359,7 @@ describe('ReactFunctionComponent', () => {
 
     expect(() => ReactTestUtils.renderIntoDocument(<Child />)).toErrorDev(
       'Each child in a list should have a unique "key" prop.\n\n' +
-        'Check the render method of `Child`.',
+        'Check the line ** in **.',
```

Comment:
You might wonder why I didn't go with the existing `Check your code at index.js:22` format we were using before. I don't think it was good. It was very easy to miss the line number in it. And beginners don't necessarily understand the `filename:line` format. So I think it's worth rearranging it.

Reply:
fwiw I think this worsens the experience for developers who do know the format. The new message requires reading and parsing the line to find the line and file. With the other format, you can pattern match the format and look for it quickly without needing to search the line for it.

Reply:
The downside of this is that auto-detection of URLs with line numbers won’t work. That’s a downside of the “local file path” format in general though. The compiled output is better in that regard because you can reliable reverse using the source map with a single set of configurations.

Reply:
My train of thought was that this is already not linkifiable because it's a single filename (as opposed to a full name with a folder or a real URL).

>With the other format, you can pattern match the format and look for it quickly without needing to search the line for it.

I might be wrong but I think you're overestimating how easy it was to see. It is easy to pattern-match when this format occurs in a predictable place (like the actual stack trace). But when it's embedded directly in the message (note none of the screenshots in this thread are showing that), I found it too easy to miss in the middle of text.

Reply:
Here is the variation with the previous format. I find it more difficult to notice there's a line number in it.

<img width="620" alt="Screenshot 2020-09-11 at 19 05 59" src="https://user-images.githubusercontent.com/810438/92958275-d887b980-f461-11ea-9d91-8c7f1de7752a.png">

Reply:
Wouldn’t it be better if it was the full url and linkified though?

Like if we for example stored a `new Error()` in the JSX call and used that stack frame if the element ends up being passed to an array. Then it’d be the same format as the rest which is source mappable.

Reply:
Yeah I was considering that but I'd expect that to be expensive. It also sucks that the browsers don't (yet) actually sourcemap it.

Reply:
This is minor, but to me “check the line 123” reads awkwardly, as opposed to “check line 123”.

Reply:
Hello, i am here =)) i love all of you <3